### PR TITLE
[luci/pass] Test for ForwardTransposeOpPass Unary ABS

### PR DESCRIPTION
**Pull request by seanshpark**
This will add unit tests for ForwardTransposeOpPass Unary ABS Op.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #10484 from draft #10485

File: compiler/luci/pass/src/ForwardTransposeOpPass.test.cpp [hunk 1]
```diff
@@ -353,3 +353,186 @@ TEST_F(ForwardTransposeToMulGraphTest, forward_transpose_mul_NEG)
...
+    _unary->x(_relu);
+
+    output()->from(_unary);
+  }
+
+protected:
+  luci::CircleRelu *_relu = nullptr;
+};
+
+void run_phase_unary(loco::Graph *g)
```
Comment (jinevening) [inline compiler/luci/pass/src/ForwardTransposeOpPass.test.cpp:464]: Do you have a reason not to use existing `run_phase`?

Comment (seanshpark) [reply compiler/luci/pass/src/ForwardTransposeOpPass.test.cpp:464]: Added `run_phase_unary()` for difference for testing but seems there was none.
I think I forgot to remove this -_-;

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**